modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointFactory.java

```diff
@@ -121,6 +122,15 @@ protected Endpoint createEndpoint(OMElement epConfig, boolean anonymousEndpoint,
                     createLoadbalanceAlgorithm(loadbalanceElement, endpoints);
             loadbalanceEndpoint.setAlgorithm(algorithm);
 
+            //set buildMessage attribute
+            String  buildMessageAtt = loadbalanceElement.getAttributeValue(new QName("buildMessage"));
```

Comment:
Same applies here. Can we use the constant defined in XMLConfigConstants?

Reply:
Done